### DISCUSSION TITLE
how to use class c in esp32

### DISCUSSION THREAD

**Discussion by manivalaguru** [Lora-net/LoRaMac-node#1090] [Q&A]
hello,
 i want to implement class c with esp32 using your libraries in am using sx1276 breakout module as my node can someone tell me how can i do that as i am a beginneer having hard time grasping this?
thanks and regards
manikandan  v

**Comment by mluis1** [comment]
The current project does not implement a port to an ESP32 MCU.

You have 2 possibilities:
1. Search on the web if someone has already done such port and use it
2. Do the port your self by following the porting guide at [Porting Guide](https://stackforce.github.io/LoRaMac-doc/LoRaMac-doc-v4.5.1/_p_o_r_t_i_n_g__g_u_i_d_e.html)

Option 2. will be more difficult and time consuming in order to get things work.

**Comment by hupiv** [comment]
Hello,

I would like just to know if someone is able to make LORAWan with esp32 ? I check for a lot of projects and didn't fine really libraries to do it ... 

Thanks
